src/profiles-tile.js

```diff
@@ -20,24 +19,24 @@ export default class ProfilesTile extends View {
 
 		this.subscriptions = new CompositeDisposable();
 
-		this.subscriptions.add(this.targetPlatform.on('click', () => {
+		// On click handler
+		this.targetPlatform.on('click', () => {
```

Comment:
Is the removal of `this.subscription.add(fn)` going to affect any functionality?